proposed/http-client/http-client.md

```diff
@@ -73,17 +73,14 @@ use Psr\Http\Message\RequestInterface;
 use Psr\Http\Client\Exception;
 
 /**
- * Exception for when a request failed, providing access to the failed request.
- *
- * This could be due to an invalid request, example when you can seek in the request body stream
- * or due invalid request object. 
+ * Exception for when a request failed.
  */
 interface RequestException extends Exception
 {
   /**
    * Returns the request.
    *
-   * The request object MAY be the same as passed to HttpClient::sendRequest()
+   * The request object MAY NOT be the same as passed to HttpClient::sendRequest()
```

Comment:
It should be a note about the immutability somewhere. I think it should be in the interface and not the PSR description because the person that would need that note will probably not go to php-fig.com to read the PSR. 

Or we could decide that a note about immutability is not needed at all..